gpflow/utilities/traversal.py

```diff
@@ -330,7 +331,7 @@ def _first_three_elements_regexp() -> Pattern[str]:
 
 
 def _str_tensor_value(value: AnyNDArray) -> str:
-    value_str = str(value)
+    value_str = str(np.around(value, 5))
```

Comment:
Not sure how valuable a test here would be - please let me know if I should add one.

Reply:
> Not sure how valuable a test here would be - please let me know if I should add one.

Why are you doing this?

Reply:
Thanks @jesnie , I had a couple of tests that failed when I was running the tests suite (locally on my m1 machine) because of string representation of some floating point values. 

The failing tests: `test_print_summary_output_string`, `test_print_summary_output_string_with_positive_minimum` had the expected `value` (e.g., in `compose_kernel_param_print_string`) of `1`, whilst the actual one was `0.99999999`. This change addresses this discrepancy.

I hope this makes sense. Thank you!